### ISSUE TITLE
Extreme Visual Glitches on Firefox with High GPU usage (Ultra 7 155H iGPU)

### ISSUE THREAD

**Issue by yippeeeyay** [IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT#869]
### Checklist [README]

- [X] Device is not undervolted nor overclocked
- [X] Device is using the [latest drivers](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/wiki/Intel-driver-updates-and-changelogs)
- [X] Application is not cracked, modded and use the latest patch

### Application [Required]

Firefox

### Processor / Processor Number [Required]

Intel Core Ultra 155H

### Graphic Card [Required]

Intel ARC Integrated Graphics

### GPU Driver Version [Required]

32.0.101.6299

### Rendering API [Required]

- [ ] Vulkan
- [ ] OpenGL
- [ ] DirectX12
- [ ] DirectX11
- [ ] DirectX10
- [ ] DirectX9
- [X] Not applicable

### Windows Build [Required]

Windows 11 24H2

### Intel System Support Utility report

[igcit_ssu.txt](https://github.com/user-attachments/files/17840952/igcit_ssu.txt)

### Description and steps to reproduce [Required]

Go to twitch.tv and start watching a stream.

https://github.com/user-attachments/assets/32a1cb6f-1c5e-402f-b6d3-e620ab0285e1

### Device / Platform

ASUS Zenbook 14 UX3405

**Comment by Felipe-Intel** [comment]
Hello @yippeeeyay our driver development team has been informed about this issue. As soon as I have news about it I'll be posting them here. In the meantime, I noticed this does not happen with Edge. You can try watching the streams in this browser in the meantime as a workaround.

**Comment by rowbawts** [comment]
Does anyone know the last known version without this issue so I can rollback?

**Comment by yippeeeyay** [comment]
> Does anyone know the last known version without this issue so I can rollback?
32.0.101.6083_101.5736
Also, I found out that turning Hardware Acceleration off on the latest drivers also fixes the graphical bugs. 
(Firefox>Settings>'Use recommended performance settings'>'Use hardware acceleration when available'

It's worth noting that even though the glitches are not on this version, scrolling while a stream is playing is extremely laggy. If you use a system with a different graphics card you can see how smooth the scrolling is supposed to be. On the Intel ARC iGPU, it jitters and lags. This is even present in Chrome, but to a much lesser degree. While watching a video on chrome on YouTube or a stream on Twitch, it will lag when you scroll while the video is playing, but very slightly. Switching the ANGLE backend in chrome://flags to DirectX 9 fixes the lag. Both Firefox and Chrome use DirectX 11 by default. Perhaps optimizing the drivers for DirectX 11 in relation to video decoding would fix the problems.

I would really appreciate if the driver development team could look into this lag problem. However, the graphics glitches are definitely higher priority.

**Comment by minchulsim** [comment]
For me I don't get visual glitches but...
My screen freezes at the last frame and sound constantly plays the last beat(?).
Laptop is not responsive at this point.
Only way to recover is press power button long enough to hard reset.

Laptop:
Ultra 5 125H with only IGPU.
Lenovo Thinkpad P14s Gen 5 Intel.

**Comment by vvvamik** [comment]
Same problem. I installed the latest driver version 32.0.101.6314 and the problem still persists.

**Comment by blami** [comment]
Hi, I have same the very same issue in Firefox (133.03). After while basically any video (e.g. youtube, twitch, or regular html5 videos on pages e.g. https://www.fractal-design.com/ or just ads) start glitching into green/black/red "noise" and CPU usage/fan goes up.

Either disabling HW Acceleration in Firefox helps or going to Intel Graphics Command Center video section and turning off all enhancements, most notably the auto-enabled ones:
* Noise Reduction
* Film Mode Detection
* Contrast Enhacements
solves the issue for me.

Laptop:
Fujitsu WU4/J3 FMV Zero (intel evo Edition)
Core Ultra 7 155H

Intel driver version: 32.0.101.6352

It does not matter whether the browser window is on internal panel or external screen connected to TB4 dock.

**Comment by vvvamik** [comment]
Even with the latest version 32.0.101.6449 the problem persists. I reverted to version 32.0.101.5972, which is without problem.

**Comment by Felipe-Intel** [comment]
Hello all, we are still working on this fix. As soon as it becomes available in a public driver I will let you know.

**Comment by SavageWaffle** [comment]
Resolution TBD

**Comment by Felipe-Intel** [comment]
@SavageWaffle this is still work in progress.

**Comment by Jorge346** [comment]
Same thing happening to my hp plus 14 with core ultra 5 125h

**Comment by bwlodarski277** [comment]
Hi, I am also having this issue with the Core Ultra 7 155H. I'm on driver version 32.0.101.6449. I see this issue on all videos in Firefox, and other browsers are fine. Firefox version 134.0.1. The device is a HP OMEN Transcend 14.

**Comment by Felipe-Intel** [comment]
> Hello [@yippeeeyay](https://github.com/yippeeeyay) our driver development team has been informed about this issue. As soon as I have news about it I'll be posting them here. In the meantime, I noticed this does not happen with Edge. You can try watching the streams in this browser in the meantime as a workaround.

Looking at [other threads](https://discord.com/channels/554824368740630529/1326628717522583652/1328406969505153125), looks like disabling hardware acceleration also does the trick. As mentioned before, we are still working on a fix.

**Comment by Fishk22** [comment]
Same issue, but I am using Chrome, and I get the same on teams conferences.

**Comment by ChristianPC2** [comment]
This issue also affects Microsoft Teams stand-alone. Videos and screensharing.

**Comment by Felipe-Intel** [comment]
@Fishk22 @ChristianPC2 If you are refering to the desktop app of Teams, please create a new ticket to report this issue with clear repro steps. Thanks!

**Comment by pro2type3** [comment]
We have this issue aswell on HP Elitebook with Intel Ultra 7 165H and U. Problem occurs in web browsers. (Chrome, Edge, Firefox) and MS Teams.

**Comment by ChristianPC2** [comment]
@Felipe-Intel When will a fix be available? Teams is not usable with any driver version newer than 32.0.101.6083_101.5736. I must disable Windows Update because otherwise the broken driver is installed again every day. This bug report is three months old!

**Comment by Felipe-Intel** [comment]
Hello all, I don't see the issue anymore with driver 101.6559. Could you please confirm on your side?

@ChristianPC2 Please check if teams issue is gone. If not, please create a new thread for better tracking.

**Comment by pro2type3** [comment]
That’s not true. Got green screen on teams meeting yesterday when host shared screen for more than 1 hour!

**Comment by yippeeeyay** [comment]
I can confirm the issue with Firefox is fixed in 101.6559.

**Comment by Felipe-Intel** [comment]
Thanks @yippeeeyay I'll proceed to close this ticket.



> That’s not true. Got green screen on teams meeting yesterday when host shared screen for more than 1 hour!

@pro2type3 Please look for any ticket created for Teams and chime in or create a new one if this problem hasn't been reported individually. Include all the information needed. This ticket was the problem with Firefox.

**Comment by AusAdvantageSean** [comment]
This issue still exists on the latest drivers. We're seeing issues in Chrome and Teams, even on 32.0.101.6732.

We've tried the WHQL drivers as well with no luck. This is on all our laptops with Intel(R) Core(TM) Ultra 5 125U

![Image](https://github.com/user-attachments/assets/b757d8c7-76d4-4aa1-99e5-a8fef7f82f83)

![Image](https://github.com/user-attachments/assets/44bc8a82-17e6-4b00-8947-c32de88cb4f7)

![Image](https://github.com/user-attachments/assets/ba1ab476-b23b-4a91-ac21-b0af5f179ae3)

**Comment by Felipe-Intel** [comment]
@AusAdvantageSean please create a new ticket and refer to this one (869). Also, check if older driver (previous WHQL, 6651 or 6647) also shows the issue. This might be a regression.

**Comment by AusAdvantageSean** [comment]
This seems silly to be creating a ticket for each individual application when it's a system wide issue. Also as mentioned, we've tried the WHQL drivers, and we had the same issues.
